<launch>
	<param name="lidar_type" type="int" value="0"/>
	<param name="blind" type="double" value="0.5"/>
	<param name="inf_bound" type="double" value="10"/>
	<param name="N_SCANS" type="int" value="1"/>
	<param name="group_size" type="int" value="8"/>
	<param name="disA" type="double" value="0.01"/>
	<param name="disB" type="double" value="0.1"/>
	<param name="p2l_ratio" type="double" value="400"/>
	<param name="limit_maxmid" type="double" value="9"/>
	<param name="limit_midmin" type="double" value="16"/>
	<param name="limit_maxmin" type="double" value="3.24"/>
	<param name="jump_up_limit" type="double" value="175.0"/>
	<param name="jump_down_limit" type="double" value="5.0"/>
	<param name="cos160" type="double" value="160.0"/>
	<param name="edgea" type="double" value="3"/>
	<param name="edgeb" type="double" value="0.05"/>
	<param name="smallp_intersect" type="double" value="172.5"/>
	<param name="smallp_ratio" type="double" value="1.2"/>
	<param name="point_filter_num" type="int" value="3"/>

	<node pkg="balm" type="livox_feature" name="livox_feature" output="screen"> 
	</node>

	<param name="root_surf_voxel_size" type="double" value="1"/>
	<param name="root_corn_voxel_size" type="double" value="1"/>
	<param name="accumulate_window" type="int" value="5"/>
	<param name="surf_filter_length" type="double" value="0.2"/>
    <param name="corn_filter_length" type="double" value="0.2"/>
	<param name="pub_skip" type="int" value="1"/>
    <node pkg="balm" type="balm_front_back" name="balm_front_back" output="screen">
	</node>

	<node pkg="rosbag" type="play" name="player" output="screen" args="/home/sam/Downloads/hall_example.bag"/>

	<arg name="rviz" default="true" />
	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find balm)/rviz_cfg/back.rviz" />
	</group>

</launch>
